src/js/DataTables/DataTable.js

```diff
@@ -225,6 +236,14 @@ export default class DataTable extends PureComponent {
       </table>
     );
 
-    return responsive ? <div className="md-data-table--responsive">{table}</div> : table;
+    return responsive ? (
```

Comment:
I'd also like the `fixed` prop for this as well. I think it should be valid to have a `responsive` table but not fixed with the additional divs.

The `DataTable` component would also pass down a `fixed` context boolean for the `TableCheckbox` and `TableColumn` components to then render the additional `div`s as well.